### DISCUSSION TITLE
Base extending downward

### DISCUSSION THREAD

**Discussion by TSims5115** [kennetek/gridfinity-rebuilt-openscad#111] [Q&A] [answered]
![image](https://github.com/kennetek/gridfinity-rebuilt-openscad/assets/133186199/7b68bc73-3b6f-4d75-8541-16dce91c0ac9)

I deleted and redownloaded all of the files but the base still looks funny.  Can anyone help?

**Comment by Ruudjhuu** [comment]
Could  you provide some more information like:

- What system are you one
- Which openscad version
- Which file are you trying to use
- What parameters did you change
- What is your goal
- What is the output of the console
- Is there output in the Error-Log

**Reply by TSims5115** [reply]
windows 10 pro

OpenSCAD version 2021.01

gridfinity-rebuild-lite

some parameters have been changed but the model looks like this on the bottom without changing anything

goal is to make the bottom of the model look like the original file

Compiling design (CSG Tree generation)...
Rendering Polygon Mesh using CGAL...
Geometries in cache: 734
Geometry cache size in bytes: 12702368
CGAL Polyhedrons in cache: 52
CGAL cache size in bytes: 103254976
Total rendering time: 0:00:00.107
   Top level object is a 3D object:
   Simple:        yes
   Vertices:     2404
   Halfedges:    8928
   Edges:        4464
   Halffacets:   4142
   Facets:       2071
   Volumes:         3
Rendering finished.

no output in error log


I just started learning openscad this is the only project ive ever used with it so far

**Reply by TSims5115** [reply, marked as answer]
> Could you provide some more information like:
> 
> * What system are you one
> * Which openscad version
> * Which file are you trying to use
> * What parameters did you change
> * What is your goal
> * What is the output of the console
> * Is there output in the Error-Log

hey thanks for trying to help.  I did resolve this issue by uninstalling openscad and reinstalling it.

**Reply by Ruudjhuu** [reply]
Good to hear it is solved.
